src/tests.rs

```diff
@@ -218,6 +218,16 @@ fn all_function_types() {
     assert_eq!(result, MyResult { value: 5 });
 }
 
+#[rstest]
+#[timeout(TEST_TIMEOUT)]
+fn handle_large_result() {
+    let initial_string = "The quick brown fox jumps over the lazy dog";
+    let result = execute_in_isolated_process(|| {
+        initial_string.repeat(10000)
+    }).unwrap();
+    assert_eq!(result.len(), 10000 * initial_string.len());
+}
+
```

Comment:
```suggestion
#[rstest]
#[timeout(TEST_TIMEOUT)]
fn handle_large_result() {
    let initial_string = "The quick brown fox jumps over the lazy dog";
    let result = execute_in_isolated_process(|| initial_string.repeat(10_000)).unwrap();
    println!("result.len(): {}", result.len());
    assert_eq!(result.len(), 10_000 * initial_string.len());
}

```

Minor formatting changes to appease `cargo fmt`.